features/steps/steps.go

```diff
@@ -147,15 +174,15 @@ func (c *Component) noEventsAreProduced() error {
 	return c.KafkaProducer.WaitNoMessageSent(c.waitEventTimeout)
 }
 
-// we are passing the string array as [xxxx,yyyy,zzz]
-// this is required to support array being used in kafka messages
-func arrayParser(raw string) (interface{}, error) {
-	// remove the starting and trailing brackets
-	str := strings.Trim(raw, "[]")
-	if str == "" {
-		return []string{}, nil
-	}
-
-	strArray := strings.Split(str, ",")
-	return strArray, nil
-}
+//// we are passing the string array as [xxxx,yyyy,zzz]
+//// this is required to support array being used in kafka messages
+// func arrayParser(raw string) (interface{}, error) {
+//	// remove the starting and trailing brackets
+//	str := strings.Trim(raw, "[]")
+//	if str == "" {
+//		return []string{}, nil
+//	}
+//
+//	strArray := strings.Split(str, ",")
+//	return strArray, nil
+// }
```

Comment:
Why?

Reply:
This has now been removed entirely

Was used for reading the godog table data but we've now switched to using a json.